zerver/webhooks/newrelic/tests.py

```diff
@@ -4,40 +4,123 @@
 class NewRelicHookTests(WebhookTestCase):
     STREAM_NAME = 'newrelic'
     URL_TEMPLATE = "/api/v1/external/newrelic?stream={stream}&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'newrelic'
 
-    def test_alert(self) -> None:
-        expected_topic = "Apdex score fell below critical level of 0.90"
-        expected_message = 'Alert opened on [application name]: Apdex score fell below critical level of 0.90 ([view alert](https://rpm.newrelc.com/accounts/[account_id]/applications/[application_id]/incidents/[incident_id])).'
+    def test_open(self) -> None:
+        expected_topic = "New Relic Alert - Test Policy (1234)"
+        expected_message = """
+Incident **opened** for condition: **Server Down** at <time:2020-11-11 22:32:11.146000>
+``` quote
+I am a test Violation
+```
+[View incident](https://alerts.newrelic.com/accounts/2941966/incidents/1234)
```

Comment:
Can we just make the link on the word `Incident` at the start of the message?  That'd be more compact without any loss of information.

Reply:
Do you think this would benefit from a topic link too?

Reply:
Generally we avoid putting links in the topic as they make them very long.